app/models/project.rb

```diff
@@ -43,6 +43,18 @@ class Project < ActiveRecord::Base
 
   scope :search, ->(name) { where("name like ?", "%#{name}%") }
 
+  def template
```

Comment:
something like:

has_one :template_stage, class_name: 'Stage', -> { where(template: true) }

On Wed, Aug 17, 2016 at 9:59 AM, Adam Panzer <notifications@github.com>
wrote:

> In app/models/project.rb
> <https://github.com/zendesk/samson/pull/1198#discussion_r75162775>:
>
> > @@ -43,6 +43,18 @@ class Project < ActiveRecord::Base
> >
> >    scope :search, ->(name) { where("name like ?", "%#{name}%") }
> >
> > +  def template
>
> Do you mean:
>
> class Project < ActiveRecord::Base
>   has_one: :template_stage
> end
>
> class TemplateStage < Stage
> end
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/zendesk/samson/pull/1198/files/66abc156d3f3ad8478ca07ee5c0e76ed494978bf#r75162775>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAAsZ3G6NOvls5jcWEiYbQQnWJ8CeY__ks5qgz3ygaJpZM4Jhtog>
> .
>